script/test_in_container/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# This Dockefile creates image where all Kompose tests can be run
+
+FROM registry.centos.org/centos/centos:7
```

Comment:
I was thinking about it. They should be build from same Dockerfile.
I don't know.

Why would you prefer image from docker hub?